keras_cv/visualization/plot_bounding_box_gallery.py

```diff
@@ -174,11 +174,4 @@ def unpackage_tfds_inputs(inputs):
             ),
         ]
 
-    plot_image_gallery(
-        plotted_images,
-        value_range,
-        legend_handles=legend_handles,
-        rows=rows,
-        cols=cols,
-        **kwargs
-    )
+    return plot_image_gallery(...)
```

Comment:
Need to include all of the params to `plot_image_gallery` -- I don't think this works unless it's some sort of Python wizardry 🧙

Reply:
I honestly don't know, but like the examples ran using this updated code😅, will return back to the one with parameters